devices/src/virtio/vhost/mod.rs

```diff
@@ -8,19 +8,19 @@
 //! Implements vhost-based virtio devices.
 
 use std;
-use sys_util::Error as SysError;
+use std::io;
 
 pub mod handle;
 pub mod vsock;
 
 #[derive(Debug)]
 pub enum Error {
     /// Creating kill eventfd failed.
-    CreateKillEventFd(SysError),
+    CreateKillEventFd(io::Error),
```

Comment:
The idea behind replacing sys_util::Error with io::Erroris is that io::Error has a to_string implementation that maps raw os error numbers to the text that you can find by looking in errno.h in the linux kernel. To benefit from this, we need to implement Debug & Display ourselves instead of relying on the defaults. For the errors from this list that generate panics, we should have a custom implementation of Debug. Take a look at the Debug implementation for Error in vmm/src/lib.rs. I expect something similar for this one as well.

Reply:
> For the errors from this list that generate panics, we should have a custom implementation of Debug. Take a look at the Debug implementation for Error in vmm/src/lib.rs. I expect something similar for this one as well.

`CreateKillEventFd`, `CloneKillEventFd`, `PollError`, `VhostIrqRead` aren't actually being used, and from what I can tell `VhostIrqCreate` doesn't cause panics.

Reply:
I left the comment at the wrong line (again). Sorry, this was meant for the Error in api_server/src/lib.rs.

Reply:
No worries; addressed the ones in `api_server/src/lib.rs`